src/evm_context.rs

```diff
@@ -405,16 +428,27 @@ pub fn encode_revert_unreachable(ssa: &mut SSATracker) -> SMTStatement {
     smt::assert(smt::neq(context().revert_flag.smt_var(ssa), 0))
 }
 
+pub fn encode_solc_panic_unreachable(ssa: &mut SSATracker) -> SMTStatement {
```

Comment:
Doesn't thin encode "reachable"? We should really take care to negate our terms at the right time...

Reply:
True